### DISCUSSION TITLE
Image recognition is broken when uploading the image, works fine when providing the path

### DISCUSSION THREAD

**Discussion by AxelFooley** [open-webui/open-webui#4903] [General]
# Bug Report
I've been trying a few vision models in the past couple of days and all of them don't work properly in open-webui.
When uploading the image the prediction is completely broken, and it seems like the LLM is rather "thinking" about a random image and providing the description of that, instead of the uploaded one.

The same model works just fine when using ollama cli.

Weirdly, if instead of uploading the image i provide the full path, then the prediction is correct.
In the screenshot below you will see the LLM going wild at the end of the message, this is not related to open-webui, using other models works just in the same way (ie. good when providing path, wrong when uploading image).

## Installation Method

Docker on Linux 

## Environment

- **Open WebUI Version:** "main" docker image
- **Ollama (if applicable):** latest windows standalone 

- **Operating System:** Ollama: Windows, open-webui: Linux
- **Browser (if applicable):** Brave

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [x ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Prediction is the same whether i provide the path or upload the image.

## Actual Behavior:

Described above

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
![chat calliope lol_c_538df3cc-0339-48d3-8cb3-8a837483fe60](https://github.com/user-attachments/assets/54f70a96-d5f5-46ba-a5eb-21925d7959b3)

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by sharoseali** [comment]
hi there. I have been facing something similar. I have written a tool in which images will be uploaded and tool will find similar image. However, when I upload the image on webui chat, the tool activates but each time it looks for the image and says please upload the image as it's not getting any images from chat. What exactly is the way to send an image to the tool?
